internal/commands/push/command.go

```diff
@@ -266,31 +266,26 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return nil
 	}
 
-	if len(appDiffs) > 0 {
+	var draft realm.AppDraft
+	if len(appDiffs) > 0 || dependenciesDiffs.Len() > 0 && hostingDiffs.Size() > 0 {
```

Comment:
I think this was my bad, but we should change this to: 

```suggestion
	if len(appDiffs) > 0 || dependenciesDiffs.Len() > 0 || hostingDiffs.Size() > 0 {
```